pallets/liquidity-pools/src/tests/inbound.rs

```diff
@@ -227,6 +235,47 @@ mod handle_tranche_tokens_transfer {
 			})
 		}
 
+		#[test]
+		fn inbound_with_frozen_investor_permissions() {
+			System::externalities().execute_with(|| {
+				config_mocks(CENTRIFUGE_DOMAIN_ADDRESS);
+				Permissions::mock_has(|_, _, _| true);
+
+				assert_noop!(
+					LiquidityPools::handle(
```

Comment:
This should be the only test in `inbound.rs` if I read this well.

Reply:
[a5f2e6e](https://github.com/centrifuge/centrifuge-chain/pull/1947/commits/a5f2e6e272c2b8400190994a4529f97ca012ad46)